src/lib.rs

```diff
@@ -78,3 +78,88 @@ pub use self::db::{Database, Error};
 #[cfg(feature = "jemalloc")]
 #[global_allocator]
 static GLOBAL: Jemalloc = Jemalloc;
+
+use std::path::PathBuf;
+
+/// Python Extension
+use pyo3::{prelude::*, wrap_pyfunction};
+use storage::SecondaryStorageOptions;
+/// Open a database for user, user can specify the path of database file
+#[pyfunction]
+fn open(path: String) -> PyResult<PythonDatabase> {
+    let runtime = tokio::runtime::Builder::new_current_thread()
```

Comment:
Is it ok to use multiple threads here?
```suggestion
    let runtime = tokio::runtime::Builder::new_multi_thread()
```

Reply:
Sure